hack/common/cne.sh

```diff
@@ -51,8 +51,8 @@ spec:
             - "--metrics-addr=127.0.0.1:9091"
             - "--store-path=/store"
             - "--transport-host=$transport_host"
-            - "--api-version=2.0"
             - "--api-port=9095"
```

Comment:
In main.go you are using api-port instead of port retrieved from transport-host, so this need to be changed to 9043